pkg/kapp/clusterapply/add_or_update_change.go

```diff
@@ -237,6 +238,9 @@ func (c AddOrUpdateChange) recordAppliedResource(savedRes ctlres.Resource) error
 		// Record last applied change on the latest version of a resource
 		latestResWithHistoryUpdated, err := latestResWithHistory.RecordLastAppliedResource(applyChange)
 		if err != nil {
+			if strings.Contains(err.Error(), "unable to record resource, limit exceed") {
```

Comment:
Is there any error code passed specifically for limit exceed? If yes, we should use that. Error messages should be the last option as they are very fragile and can be changed over time.

Reply:
This is just for internal purposes, not something shown to the user. I believe `508` error code is used for `Resource Limit Reached`  but looks like we don't use the error code in our cli.